dex/asset.go

```diff
@@ -29,7 +29,7 @@ const Simnet = Regtest
 // through the provided logger.
 type Logger = slog.Logger
 
-// Type Asset combines the DEXAsset backend with the configurable asset
+// Type Asset combines the Backend with the configurable asset
```

Comment:
The language "combines the Backend" seems out of place, like it was meant for asset.BackedAsset, which does combine a `dex.Asset` (this) with a `DEXAsset`.